staging/src/k8s.io/kubectl/pkg/cmd/diff/diff.go

```diff
@@ -56,7 +56,13 @@ var (
 
 		KUBECTL_EXTERNAL_DIFF environment variable can be used to select your own
 		diff command. By default, the "diff" command available in your path will be
-		run with "-u" (unified diff) and "-N" (treat absent files as empty) options.`))
+		run with "-u" (unified diff) and "-N" (treat absent files as empty) options.
+		
+		If the exit code after diff command is 0, means there is no change detected 
```

Comment:
This is the part that you should replace.

Reply:
Ok, done!